### PR TITLE
H wadd spring

### PR DESCRIPTION
**Pull request by margarita-ignatieva**

### CODE REVIEW

File: src/main/java/com/cinema/shop/dao/impl/TicketDaoImpl.java [hunk 1]
```diff
@@ -2,24 +2,30 @@
...
 import org.apache.log4j.Logger;
 import org.hibernate.Session;
+import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
+import org.springframework.stereotype.Repository;
 
-@Dao
+@Repository
 public class TicketDaoImpl implements TicketDao {
     private static final Logger log = Logger.getLogger(TicketDaoImpl.class);
```
Comment (KovalAlexey) [inline src/main/java/com/cinema/shop/dao/impl/TicketDaoImpl.java]: you can remove this empty line

File: src/main/java/com/cinema/shop/dao/impl/UserDaoImpl.java [hunk 2]
```diff
@@ -2,26 +2,32 @@
...
 import org.hibernate.Session;
+import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
+import org.springframework.stereotype.Repository;
 
-@Dao
+@Repository
 public class UserDaoImpl implements UserDao {
     private static final Logger log = Logger.getLogger(UserDaoImpl.class);
```
Comment (KovalAlexey) [inline src/main/java/com/cinema/shop/dao/impl/UserDaoImpl.java]: same here and through the project

File: src/main/java/com/cinema/shop/config/WebConfig.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"com.cinema.shop.controllers"})
+public class WebConfig {
+
```
Comment (DmGrt) [inline src/main/java/com/cinema/shop/config/WebConfig.java]: Redundant line)

**Review by KovalAlexey — APPROVED** [review]

File: src/main/java/com/cinema/shop/service/impl/AuthenticationServiceImpl.java [hunk 4]
```diff
@@ -1,21 +1,27 @@
...
 public class AuthenticationServiceImpl implements AuthenticationService {
-    @Inject
-    private UserService userService;
-    @Inject
-    private ShoppingCartService shoppingCartService;
+
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+
+    @Autowired
```
Comment (DmGrt) [inline src/main/java/com/cinema/shop/service/impl/AuthenticationServiceImpl.java]: I think this annotation is redundant here :) and same in other classes

**Review by DmGrt — COMMENTED** [review]
Good job!

File: src/main/java/com/cinema/shop/service/impl/AuthenticationServiceImpl.java (hunk 4 above)
Comment (margarita-ignatieva) [reply src/main/java/com/cinema/shop/service/impl/AuthenticationServiceImpl.java]: just to be 100500% sure everything will work fine, but you-re right

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**